src/go/k8s/pkg/resources/configmap.go

```diff
@@ -41,6 +42,10 @@ const (
 
 	oneMB          = 1024 * 1024
 	logSegmentSize = 512 * oneMB
+
+	scramUsername       = "pandaproxy_client"
```

Comment:
NIT: @Mateoc @andresaristizabal @senior7515 this is hidden user that need to be caught in UI or API so that our end user can not create service account `pandaproxy_client`

Reply:
good point